src/main/java/org/assertj/core/api/AbstractMapAssert.java

```diff
@@ -1499,12 +1499,14 @@ public SELF withThreadDumpOnError() {
    * If a given key is not present in the map under test, a null value is extracted.
    * <p>
    * Example:
-   * <pre><code class='java'> Map&lt;String, Object&gt; map = new HashMap&lt;&gt;();
+   * <pre><code class='java'>
+   * Map&lt;String, Object&gt; map = new HashMap&lt;&gt;();
```

Comment:
please keep the original formatting

Reply:
Thanks for the feedback! I gave a newline to the code snippet because the rendering on IntelliJ aligns the first line wrongly:

![Screenshot 2019-04-07 at 10 33 12](https://user-images.githubusercontent.com/26772046/55680943-e22df900-5920-11e9-9755-99996242d443.png)

But it looks like [it doesn't hurt on html javadoc](http://joel-costigliola.github.io/assertj/core-8/api/org/assertj/core/api/AbstractMapAssert.html#extracting-java.lang.Object...-).

Shall I revert those lines back then? I guess if you want to address this topic, it's probably best to do it in a separate PR to have a consistent style everywhere.

Reply:
alright then, I remember doing this it used to add new lines but that might have changed